smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/package-info.java

```diff
@@ -80,6 +80,10 @@
  * <td>XMPP service to run the tests on</td>
  * </tr>
  * <tr>
+ * <td>Name (DNS name or IP address) of the host providing the XMPP service.</td>
```

Comment:
That seems wrong, shouldn't there be the name of the configuration option?

```suggestion
 * <td>host</td>
```

Reply:
Thanks, got this change fixed up with the previous one.